src/orange/OrangeTrackView.hh

```diff
@@ -144,13 +142,20 @@ class OrangeTrackView
     // Create local distance
     inline CELER_FUNCTION detail::TempNextFace make_temp_next() const;
 
-    inline CELER_FUNCTION detail::LocalState make_local_state() const;
+    inline CELER_FUNCTION detail::LocalState
+    make_local_state(LevelId level) const;
 
     // Whether the next distance-to-boundary has been found
     CELER_FORCEINLINE_FUNCTION bool has_next_step() const;
 
     // Invalidate the next distance-to-boundary
     CELER_FORCEINLINE_FUNCTION void clear_next_step();
+
+    // Make a LevelStateAccessor for the current thread and level
+    CELER_FORCEINLINE_FUNCTION LevelStateAccessor make_lsa() const;
+
+    // Make a LevelStateAccessor for the current thread and a given level
+    CELER_FORCEINLINE_FUNCTION LevelStateAccessor make_lsa(LevelId level) const;
```

Comment:
Minor change for a code grouping standpoint: can you move these to line 147 above (so they're next to the other `make` functions)?